Content/src/Server/paket.references

```diff
@@ -9,11 +10,15 @@ group Server
 //#elseif (Server == "saturn")
   Saturn
 //#endif
+
+//#if     (!Remoting && Server != "suave")
+  Fable.JsonConverter
+//#endif  
+
 //#if     (Remoting && Server == "suave")
   Fable.Remoting.Suave
-//#elseif (Remoting && Server == "giraffe")
-  Fable.Remoting.Giraffe
-//#elseif (Remoting && Server == "saturn")
+//#elseif (Remoting)
```

Comment:
This removes a warning during template generation that added the same dependency twice.